components/developer/jenkins-core-weekly/files/Makefile

```diff
@@ -1,4 +1,19 @@
-JENKINS_VERSION ?= 2.65
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016-2017 Jim Klimov
+#
+
+JENKINS_VERSION ?= 2.70
```

Comment:
Could we make this value go away and read from somewhere? It's not nice if you have to bump in two different places.